### ISSUE TITLE
This repo is abandoned. What a mess. How to move forward?

### ISSUE THREAD

**Issue by peacechen** [d-a-n/react-native-modal-picker#63]
There are tons of forks & PRs that have been pending for this project.  How do we migrate this to an active repo where more than one person can merge and publish?  Something like [react-community](https://github.com/react-community) ?

Ideally if we could get hold of @d-a-n , he could add others as [Collaborators](https://help.github.com/articles/permission-levels-for-a-user-account-repository/#collaborator-access-on-a-repository-owned-by-a-user-account) so they can commit and merge. 
Also add them as collaborators to npm to publish new versions.

**Comment by AlexisJamin** [comment]
Yes, agree ! Where are you @d-a-n  ?

**Comment by matteo-pennisi** [comment]
@peacechen Right now we can only make a fork and try to migrate all the dev there. If you want to do this, make a new issue with a noticeable tag like **[MIGRATED DEVELOPING]** so anyone will know :wink:

**Comment by peacechen** [comment]
@matteo-pennisi  Good idea.  I'll create another thread with details about the migration.
